### DISCUSSION TITLE
i dont get error when i signin with wrong infos! why?

### DISCUSSION THREAD

**Discussion by saman1324** [supabase/supabase#5959] [Questions] [answered]
hello.
i was testing my web app and i found a strange problem in login form. when i login wrong infos, i dont get an error about that! look at this example:
```
let { user, registerationError } = await supabase.auth.signIn({
    email: 'someon',
    password: 'MpcJntIzCiJe'
})
```
with this code, i should get some information about the error in `registerationError`. but i dont! even when i enter an email and password that are not in athentication section, i just get error 400 in the console. it wasnt like this before. i finished my login form and it worked correctly but now, it doesnt.
ill be happy to hear your solution.

**Comment by GaryAustin1** [comment, marked as answer]
You would need to alias like this  let {user, error:registerationError} = ....

**Reply by saman1324** [reply]
thank you! what a bad mistake... look at me that i thought the problem is with supabase. im laughing at my mistake🤣🤣🤣
i was coding for 3 hours and i know you know it is normal...
